webdriver-spec.html

```diff
@@ -3228,7 +3228,7 @@ <h2 id=element-displayedness>Element Displayedness</h2>
   </section>
 </section>
 <section>
-  <h2>Executing Javascript</h2>
+  <h2>Executing Script</h2>
   <div class="note">
```

Comment:
I didn’t complain about the naming of the command.  There are some cases of lowercased "javascript" elsewhere.  I think "script" is better, for the reasons you mention.